src/framework/theme/components/chat/chat-message-quote.component.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NbChatModule, NbThemeModule } from '@nebular/theme';
+
+@Component({
+  selector: 'nb-chat-message-quote-test',
+  template: `
+        <nb-chat-message-quote
+            [sender]="sender" [date]="date" [dateFormat]="dateFormat"
+            [message]="message" [quote]="quote">
+        </nb-chat-message-quote>
```

Comment:
Spacing (4 -> 2). Try to run format command in the your editor (`Cmp + Shift + P` -> `Format document`) to fix all missed lines automatically.